routes.yaml

```diff
@@ -3,4 +3,4 @@ group: api
 weight: 991
 routes:
    1: ^/psc-statement-data-api/healthcheck
-   2: ^/company/(.){8}/persons-with-significant-control-statement/(.*)
\ No newline at end of file
+   2: ^/company/(.){10}/persons-with-significant-control-statements
```

Comment:
I think this needs to be 8-10, there's a couple of edge cases. Also do we need a new 3rd route for the other requests with a /.*?